src/lightning/app/core/api.py

```diff
@@ -410,21 +413,21 @@ async def websocket_endpoint(websocket: WebSocket):
     await websocket.close()
 
 
-async def api_catch_all(request: Request, full_path: str):
+async def api_catch_all(request: Request, full_path: str) -> None:
     raise HTTPException(status_code=404, detail="Not found")
 
 
 # Serve frontend from a static directory using FastAPI
 fastapi_service.mount("/static", StaticFiles(directory=frontend_static_dir, check_dir=False), name="static")
 
 
-async def frontend_route(request: Request, full_path: str):
+async def frontend_route(request: Request, full_path: str):  # type: ignore[no-untyped-def]
```

Comment:
this looks like Union[str, templates.TemplateResponse]

Reply:
tried and do not like it because of the template